packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAPPaymentQueueDelegate.h

```diff
@@ -13,8 +13,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-API_AVAILABLE(ios(13))
-API_UNAVAILABLE(tvos, macos, watchos)
+API_AVAILABLE(ios(13), macos(10.15))
```

Comment:
This should've always been here. When mac support first got added i think this got missed [here](https://github.com/flutter/packages/blob/23d5bbbefa285986839a3654b868f06429b185cd/packages/in_app_purchase/in_app_purchase_storekit/shared/Classes/FIAPPaymentQueueDelegate.h)
For some reason the objc macos tests still ran fine even though it was technically unavailable, but when i rewrote the tests in swift, the compiler complained.
Adding this should be fine as its a wrapper over [SKPaymentQueueDelegate](https://developer.apple.com/documentation/storekit/skpaymentqueuedelegate) and mac is supported